src/server/twemcache-rs/src/main.rs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2021 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+#[macro_use]
+extern crate rustcommon_logger;
+
+use backtrace::Backtrace;
+use pelikan_twemcache_rs::TwemcacheBuilder;
+
+use rustcommon_logger::{Level, Logger};
+
+fn main() {
+    // custom panic hook to terminate whole process after unwinding
+    std::panic::set_hook(Box::new(|s| {
+        error!("{}", s);
+        println!("{:?}", Backtrace::new());
+        std::process::exit(101);
+    }));
+
+    // initialize logging
+    Logger::new()
+        .label(env!("CARGO_CRATE_NAME"))
+        .level(Level::Info)
```

Comment:
How do I override this in config?

Reply:
Currently, you can't. Will fix

Reply:
Fixed this by lifting the config parsing up from the library to the binary target. I've made minor changes to the config crate so that we're using human readable log levels and not numeric levels - I think it winds up being easier from a config perspective.